src/OpenFeature/OpenFeature.csproj

```
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVer)" />
```

Is it possible to limit this package to the frameworks that need it as you mention in the PR description?
```suggestion
    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVer) Condition="'$(TargetFramework)' == 'netstandard2.0'"" />
```

---

It should be, let me try that locally!

---

Yep, works as expected, thanks for the suggestion!

---

Nice call @askpt